app/services/session_encryptor.rb

```diff
@@ -8,15 +8,13 @@ def self.build_user_access_key
     build_user_access_key
   end
 
-  # rubocop:disable Security/MarshalLoad
   def self.load(value)
     decrypted = encryptor.decrypt(value, user_access_key)
-    Marshal.load(::Base64.decode64(decrypted))
+    ActiveSupport::JSON.decode(::Base64.decode64(decrypted))
```

Comment:
What if we use `.with_indifferent_access` here, so we don't need to change the entire app to use strings instead of symbols?